cedar-policy-validator/src/schema.rs

```diff
@@ -474,7 +474,7 @@ impl ValidatorSchema {
             for parent in entity_type.parents.iter() {
                 entity_children
                     .entry(internal_name_to_entity_type(parent.clone())?)
-                    .or_insert_with(HashSet::new)
+                    .or_default()
```

Comment:
Can you check if this is actually the same? I can imagine that `or_default` could reasonably not insert default value

Reply:
It seems `or_default` does the job: https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html#method.or_default